docs/how_to_migrate_from_2.2_to_2.3.md

```diff
@@ -292,6 +293,16 @@ these values to the module of your controller like this:
 </script>
 ```
 
+### Adapt paths for 'less'
+
+We no more load all `.less` files in each application. You must import your `<interface.less>` in each
+application controller you have by somthing like: `import '../../less/<interface>.less'`.
```

Comment:
`We no more load all `.less` files in each application. ` it was already the case...

Reply:
`something` instead of `somthing`